### DISCUSSION TITLE
What CIS Versions are used?

### DISCUSSION THREAD

**Discussion by paparooky** [usnistgov/macos_security#181] [Q&A] [answered]
Looking at the baselines, there is no indication of which version of each CIS OS baseline is being used. For example, for 10.15 Catalina: CIS has v2.0.0 and 2.1.0 available, Big Sur has the same, Monterey has v1.0.0 and v1.1.0. Are the baselines in the project updated to the most recent CIS baselines?

Also, in the Catalina branch, only cisv8 is available but in Big Sur, Monterey, and Ventura cis_lvl1, cis_lvl2, and cisv8 are available. Are the lvl1 and lvl2 baselines older v7 CIS controls?

**Comment by robertgendler** [comment, marked as answer]
The benchmark version is in the baseline yaml file. 

It was decided that we would not include the Catalina benchmark in the project when we included the benchmarks.

**Reply by paparooky** [reply]
Thank you. Where does CIS publish the Ventura baseline?

**Reply by robertgendler** [reply]
They haven't yet. Ventura is still under development

**Reply by paparooky** [reply]
Right. So for the Ventura branch are you just using the Monterey controls or is there some kind of preview that you have access to?

Back to the versions, I see in the lvl1 and lvl2 yaml files that there is versioning information, in the v8 baselines there is not. Are those just the latest versions?

For Context: I am trying to compare the compliance reports mSCP generates to the reports generated by Nessus scans. But I want to make sure I am doing an apples to apples comparison.

**Reply by robertgendler** [reply]
Cis v8 is like CIS's 800-53.  It's a set of controls. Your organization makes the risk based decision what to apply and the organizationally defined values for those controls.  

The cis benchmark is a defined list of settings and values they suggest.  

The CIS benchmark is open for anyone to view, even ones under development. You just need to register for CIS workbench. 

But I believe currently the baseline files in the project are just the Monterey benchmarks migrated over to Ventura.

**Reply by paparooky** [reply]
Got it, that all makes sense.

I guess what I am hung up on is my understanding of v8 controls is they are generic controls that are applicable to any system. The baselines then take those controls and make them specific to a system, OS, etc. However, if I run the generate guidance script with the cisv8.yaml in Catalina, I still get a very specific list of commands and settings to not only determine compliance but to also run commands that will bring the system up to compliance. So the cisv8.yaml is acting like a baseline as far as I understand. Am I missing something?

**Reply by robertgendler** [reply]
Much like 800-53, we've translated CISv8 control to a setting within the macOS. The CIS v8 controls are generic controls. Your organization can interpret them however you'd like. We've interpreted the control to setting mapping with what we think they equal as well as general best practices when it comes to values for settings.

cisv8.yaml is a baseline. You are able to customize it to your hearts content. 
the cis benchmarks are baselines that have been decided by CIS, this is how to meet what they recommend as their best practices.

**Reply by paparooky** [reply]
Got it, so the cisv8.yaml is mSCP's (and NIST's after SP800-219?) interpretation of CISv8 controls for macOS [insert version here] while the cis_lvl1.yaml and the cis_lvl2.yaml are CIS's official baseline guidance based on how much hardening you want to do.

Am I on the right track now?

**Reply by robertgendler** [reply]
Correct.

The only change in what you've said cis_lvl1 and lvl2 are benchmarks. Small change but makes a difference in the meaning.

Benchmarks are defined set of settings and values.

**Reply by paparooky** [reply]
Appreciate you sticking with me. Thanks!
